metroae

```diff
@@ -642,13 +644,24 @@ function audit_log_and_exit {
     print_version_and_exit $1
 }
 
+function setup_log_files {
```

Comment:
Let's split this function into two separate ones: create_log_files and setup_log_files. In create_log_files, we will:
1. create the logs directory
2. create the $ANSIBLE_LOG file
3. create the $AUDIT_LOG file

In setup_log_files, we will:
1. set the current time
2. create the symlink between $ANSIBLE_LOG and the timestamped file